### ISSUE TITLE
amp-video-docking: Visual tests failing

### ISSUE THREAD

**Issue by choumx** [ampproject/amphtml#28734]
@alanorozco says this code didn't change recently so I'm not sure if this is a one-off or actual problem.

> The CSS selector .i-amphtml-video-interface does not match any elements in the page

https://percy.io/ampproject/amphtml/builds/5576997

**Comment by choumx** [comment]
This might be a real error/bug actually.

**Comment by choumx** [comment]
Actually this is caused by amp-video-iframe failing to build. This happens because `intersect-resources` causes it to be built before measure, and a zero-size layout box triggers `looksLikeTrackingIframe()`. Lucky catch! :)
